app/modules/Home/components/HomeComponent.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { type FC } from 'react';
+import { View, Text, FlatList } from 'react-native';
+import { CustomCard, CustomDropDown } from '../../../components';
+import type { CustomCardPropType } from '../../../components/CustomCard/Types';
```

Comment:
import from components index.ts file

Reply:
@Krunal-K-SimformSolutions For this I want to connect.